saskatoon/sitebase/templates/app/detail_views/property/harvest_row.html

```diff
@@ -1,7 +1,37 @@
 {% load i18n %}
 {% load static %}
 <tr>
-    <td>{{ harvest.pick_leader__person__first_name }}</td>
+    <td>
+        <a href='{% url "harvest-detail" harvest.id %}'>#{{ harvest.id }} Harvest by {{ harvest.pick_leader.name }}</a>
```

Comment:
@mina-andajos same here, keep in mind Orphan harvests don't have a pick_leader yet.

Reply:
@2ynn I believe it is fixed now 👍

Reply:
can you submit a pull request to fix for reverse urls issues?